### ISSUE TITLE
The React Europe Conference source does not seem web ready

### ISSUE THREAD

**Issue by SynerG** [EvanBacon/AWESOME-REACT-NATIVE-WEB#5]
The 'React Europe Conference App' website works fine and looks like a great example of RNW/Expo Web development.
However, the source code linked in the repo does not seem web-ready.
Its `app.json` only mentions `android` and `ios`, not `web`.
Cloning that repo, installing dependencies and running `expo start --web` does not work for me. It spits multiple errors as it tries to bundle react-native libraries.

I guess that the web version had changes that are not committed to the github repo.

It is a pity as I was trying to find an example of a comprehensive RN web app (not a game) that included navigation. 
(The Instagram clone would also be a good example, but does not include source code).

I would open an issue in that repo asking if they could commit the changes to make it web compatible, but the repo does not allow to open issues.

Are you aware of how to run the source code on web? Maybe the readme of this repo could include some tips

**Comment by EvanBacon** [comment]
> I guess that the web version had changes that are not committed to the github repo.

Right, I need to PR the changes in. Unfortunately things are changing a lot with Expo web beta (which makes most of this possible).

> (The Instagram clone would also be a good example, but does not include source code).

Will publish.

>  (not a game) that included navigation.

Lol sorry 😅 please feel free to find and link some, I only know about them if someone tells me.
